Px.Search/Indexer.cs

```diff
@@ -101,7 +109,22 @@ private void TraverseDatabase(string id, string language, IIndex index)
                     }
                     else if (subitem is TableLink)
                     {
-                        IndexTable(((TableLink)subitem).TableId, (TableLink)subitem, language, index);
+                        string tableId = ((TableLink)subitem).TableId;
+                        if (!_indexedTables.Contains(tableId))
+                        {
+                            IndexTable(tableId, (TableLink)subitem, language, index);
+
+                            _indexedTables.Add(tableId);
+                            if (_indexedTables.Count % 100 == 0)
+                            {
+                                _logger.LogInformation($"Indexed {_indexedTables.Count} tables ...");
+                            }
+                        }
+                        else
+                        {
+                            _logger.LogWarning($"Table {tableId} is already indexed.");

```

Comment:
Should not be a warning is expected that a table can be in multiple places in the database. Make it as Information or Debug

Reply:
Yes, thank you.